packages/liveblocks-node/src/client.ts

```diff
@@ -69,36 +117,84 @@ export class Liveblocks {
   }
 
   /** @internal */
-  private async post(
-    path: `/${string}`,
-    json: Record<string, unknown>
-  ): Promise<Response> {
+  private async post(path: URLSafeString, json: Json): Promise<Response> {
```

Comment:
Updated both `post` and `put` methods to accept json of type `Json`. This required updating the types for a few other things that relied on this method, e.g., `userInfo` in the `CreateSessionOptions` type that is used by `prepareSession` method was previously typed as `unknown` but is now typed as `Json`.

https://github.com/liveblocks/liveblocks/blob/270650251d1f7af389427ac866047715a370f91a/packages/liveblocks-node/src/Session.ts#L100

Reply:
Seeing this reminds me that we should probably also stricten the input type for `_userInfo` in TypeScript here too. I just made a backend change that will get deployed soonish, where the shape of the passed-in `userInfo` will be enforced to be of [`IUserInfo`](https://github.com/liveblocks/liveblocks/blob/main/packages/liveblocks-core/src/protocol/BaseUserMeta.ts#L3-L12) type (which is a strict subset of JSON).

Specifically:
- It's a JSON object
- If that object has a `name` field, it _must_ be a string
- If that object has a `avatar` field, it _must_ be a string
- All other fields are free to use and can be free-form JSON

Since the backend is going to enforce that shape soon, it may be good to include this strictening of this input type here with your changes as well.